hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -416,6 +418,8 @@ ContainerCommandResponseProto handleDeleteContainer(
       deleteInternal(kvContainer, forceDelete);
     } catch (StorageContainerException ex) {
       return ContainerUtils.logAndReturnError(LOG, ex, request);
+    } catch (IOException ex) {
```

Comment:
This IOException comes from [here](https://github.com/apache/ozone/blob/78447eb08463a89a2ad0c75d5ce0b2e3597705a6/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/KeyValueContainerUtil.java#L568). I don't think we should surpress this or change it to `StorageContainerException`.

Reply:
@errose28 Just logged the error and removed the exception alltogether, since you mentioned that container and volume scans will deal with it.

Reply:
We still need to throw the StorageContainerException if the move fails so that SCM will retry the delete. My point about the scanners was that it is ok if SCM keeps retrying even though there is some IO issue with the volume, because the scanners will eventually pick up on it and fail the volume, after which the deletions will not be sent again.